packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -760,10 +803,8 @@ class SizeObserver extends OneChildRenderObjectWidget {
 }
 
 
-// CONVENIENCE CLASS TO COMBINE COMMON PAINTING, POSITIONING, AND SIZING NODES
```

Comment:
This file is split into sections with all-caps headings... we should either get rid of all of them or make sure they're all consistent.